### PR TITLE
Updated wording deposit reclaim

### PR DESCRIPTION
**Pull request by setbern**
Text updates for the reclaim:

On the "Deposit - processing" page
After 6 blocks, if a deposit hasn't cleared, add language that says
There is a delay in processing your deposit, but signers are still working on it and your funds are secure. Please contact our support team here to help us expedite it.

image

2. On the "Reclaim" page
On the reclaim page itself add additional language that says:
Please note that we have received reports of errors with Ledgers running the reclaim function - if you are using a ledger and experience this please contact our support team here

image

### CODE REVIEW

File: src/comps/Deposit.tsx [hunk 1]
```diff
@@ -479,6 +497,23 @@ const DepositFlowReview = ({ txId }: DepositFlowReviewProps) => {
...
+        {showDepositWarning && (
+          <div className="flex flex-1 items-end">
+            <SubText>
+              There is a delay in processing your deposit, but signers are still
+              working on it and your funds are secure. Please contact our
+              support team
+              <Link
+                className="text-blue-500 underline"
+                href="https://direct.lc.chat/18945692/"
+              >
```
Comment (hozzjss) [inline src/comps/Deposit.tsx:509]: this should be read from env

File: src/comps/ReclaimManager.tsx [hunk 2]
```diff
@@ -453,6 +454,20 @@ const ReclaimDeposit = ({
...
           </div>
+          <div className="flex flex-1 items-end">
+            <SubText>
+              Please note that we have received reports of errors with Ledgers
+              running the reclaim function - if you are using a ledger and
+              experience this please contact our support team
+              <Link
+                className="text-blue-500 underline"
+                href="https://direct.lc.chat/18945692/"
+              >
```
Comment (hozzjss) [inline src/comps/ReclaimManager.tsx:465]: read LIVE_CHAT_ID from env here

File: src/hooks/use-deposit-status.ts [hunk 3]
```diff
@@ -81,8 +86,14 @@ export function useDepositStatus(txId: string) {
             return;
           }
           const currentBlockHeight = await getCurrentBlockHeight();
+
+          setCurrentBlockHeight(currentBlockHeight);
+
+          const confirmedBlockTime = info.status.block_height || 0;
+          setConfirmedBlockHeight(confirmedBlockTime);
```
Comment (hozzjss) [inline src/hooks/use-deposit-status.ts:93]: this can be inferred from set status response no need for a new state

File: src/comps/Deposit.tsx [hunk 4]
```diff
@@ -479,6 +498,25 @@ const DepositFlowReview = ({ txId }: DepositFlowReviewProps) => {
...
+          <div className="flex flex-1 items-end">
+            <SubText>
+              There is a delay in processing your deposit, but signers are still
+              working on it and your funds are secure. Please contact our
+              support team
+              <a
+                target="_blank"
+                rel="noreferrer"
+                className="text-blue-500 underline"
+                href={`https://direct.lc.chat/${bridgeConfig.LIVECHAT_ID}/`}
```
Comment (aldur) [inline src/comps/Deposit.tsx:511]: Don't we need to gave this if `LIVECHAT_ID` is not set?

File: src/comps/ReclaimManager.tsx [hunk 5]
```diff
@@ -453,6 +454,22 @@ const ReclaimDeposit = ({
...
+          <div className="flex flex-1 items-end">
+            <SubText>
+              Please note that we have received reports of errors with Ledgers
+              running the reclaim function - if you are using a ledger and
+              experience this please contact our support team
+              <a
+                target="_blank"
+                rel="noreferrer"
+                className="text-blue-500 underline"
+                href={`https://direct.lc.chat/${LIVECHAT_ID}/`}
```
Comment (aldur) [inline src/comps/ReclaimManager.tsx:466]: Same here, what if `LIVECHAT_ID` is empty?

File: src/comps/ReclaimManager.tsx [hunk 6]
```diff
@@ -453,6 +454,22 @@ const ReclaimDeposit = ({
               {useShortAddress(getWalletAddress() || "")}
             </p>
           </div>
+          <div className="flex flex-1 items-end">
+            <SubText>
+              Please note that we have received reports of errors with Ledgers
+              running the reclaim function - if you are using a ledger and
+              experience this please contact our support team
+              <a
```
Comment (aldur) [inline src/comps/ReclaimManager.tsx:462]: Nit: not sure if just in the link, but it looks like the screen includes spaces. Can you check/fix it? 
Screenshot 2024-12-20 at 12 09 25 PM

**Pull request merged**